pom.xml

```diff
@@ -86,6 +86,16 @@
 			<version>0.11.1</version>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
```

Comment:
Had never heard of shedlock before but makes sense if we ever want to load balance the application.